react-state/my-app/src/ImageCaption.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+type Props = {
+  imageCap: string[];
+};
+
+export function ImageCaption({ imageCap }: Props) {
+  const [caption, changeCaption] = useState(0);
```

Comment:
The React convention is to name state setters starting with `set`. It's just a convention, but it would be good to get in that habit.